testing/test_windows_subst.py

```diff
@@ -0,0 +1,95 @@
+import os.path
+import subprocess
+import sys
+from string import ascii_lowercase
+
+from py._path.local import LocalPath
+
+import pytest
+from _pytest import pytester
+from _pytest.fixtures import FixtureRequest
+
+
+class _SubstTmpdirFactory:
+    """
+    A temp dir factory which may do a subst on Windows drives.
+    """
+
+    def __init__(self, tmpdir, tmpdir_factory, subst):
+        self.tmpdir = tmpdir
+        self.tmpdir_factory = tmpdir_factory
+        self.subst = subst
+        self.subst_drives = {}
+
+    def _subst_path(self, filename):
```

Comment:
This is actually a directory name, right? If so I suggest to rename this:

```suggestion
    def _subst_path(self, directory):
```